crates/holochain/tests/tests/mod.rs

```diff
@@ -3,6 +3,7 @@ mod agent_scaling;
 mod app_interface_security;
 mod authored_test;
 mod clone_cell;
+mod conductor;
```

Comment:
I prefer a more specific name like
```suggestion
mod app_installation;
```